massgen/backend/antigravity_cli.py

```
@@ -0,0 +1,784 @@
+                await proc.wait()
+                await queue.put(_DONE)
+
+        async def _tail() -> None:
+            try:
+                async for chunk in self._find_and_tail_transcript(brain_dir, pre_existing, proc):
+                    produced_real_output["value"] = True
+                    await queue.put(chunk)
+            finally:
+                await queue.put(_DONE)
+
+        asyncio.create_task(_read_stdout())
```

Ruff (0.15.13)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)